src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -1189,5 +1196,21 @@ static class BatchElimination {
 
   static class TabulationCancelledException extends Exception {
 
+    final boolean cancelledByUser;
```

Comment:
this doesn't appear to be used

Reply:
Am I misunderstanding your comment? It's used just below.

Reply:
Ah, I see - you did it this way so that you didn't have to update existing usages. Can we please remove the default constructor so that all usages must be explicit about this?